frontend/src/MovieGallery/component/Home.tsx

```diff
@@ -0,0 +1,56 @@
+import UpcomingMovieGallery from "./UpcomingMovieGallery";
+import {useEffect, useState} from "react";
+import {UpcomingMovie} from "../model/UpcomingMovie";
+import axios from "axios";
+import TopRatedMovieGallery from "./TopRatedMovieGallery";
+import {TopRatedMovie} from "../model/TopRatedMovie";
+import {PopularMovie} from "../model/PopularMovie";
+import PopularMovieGallery from "./PopularMovieGallery";
+
+export default function Home(){
+
+    const [upcomingMovies, setUpcomingMovies] = useState<UpcomingMovie[]>([])
```

Comment:
Looks like you have a lot of code duplications for almost the same movies, maybe you could optimise/combine that a bit 👍